client/src/components/DataFiles/DataFilesProjectMembers/_cells/SystemRoleSelector.jsx

```diff
@@ -88,9 +88,6 @@ const SystemRoleSelector = ({ projectId, username }) => {
         onChange={(e) => setSelectedRole(e.target.value)}
         className={styles['project-role-selector']}
       >
-        {username !== authenticatedUser && (
-          <option value="ADMIN">Administrator</option>
-        )}
         <option value="USER">User (read/write)</option>
```

Comment:
Not this file, but when viewing team members that aren't in the portal database, we can grab their user info from Tapis via `client.authenticator.get_profile(username=username)`, which includes name, email, and username data
![Screen Shot 2023-03-14 at 3 57 04 PM](https://user-images.githubusercontent.com/20326896/225134826-205f4188-48b7-400c-b5b2-b23ef49ebb5e.png)

Reply:
Is there a use case for viewing team members that aren't in the db? Users can only be added to a workspace through the UI if they have an associated Django user model. Also, is any random user able to look up any other user's profile info?

Reply:
It's possible A2CPS systems may have non-portal users. And yeah, any user can look up profile info apparently, which I also found surprising

Reply:
I almost don't want to rely on that behavior since it seems like a loophole that might close in the future.

Reply:
I'm remembering now that they did handle Shared Workspace permissions out of band in a2cps, i.e. created systems and shared them directly with users outside of the portal. I think it'd be good to handle this case one way or another (TAS or Tapipy), so we at least show something that doesn't look like `sal ()`

Reply:
Or at the very least just show `sal`, without the parens

Reply:
For sure. I'm wary of sticking a TAS call in there just because there's the potential for us to end up with a ton of synchronous API calls in the "project detail" view, which would cause long wait times for users.

Reply:
Refactored to get rid of weird formatting/parentheses when the user isn't in the db.

Reply:
Sounds good 👍